doc/md/language-manual.md

```diff
@@ -1264,7 +1272,10 @@ The argument type of `inspect` depends on the interface of the enclosing actor.
 
 -   `arg : Blob`: the raw, binary content of the message argument;
 
--   `msg : <variant>`: a variant of *decoding* functions, where `<variant> == {…​; #<id>: () → T; …​}` contains one variant per shared function, `<id>`, of the actor. The variant’s tag identifies the function to be called; The variant’s argument is a function that, when applied, returns the (decoded) argument of the call as a value of type `T`.
+-   `msg : <variant>`: a variant of *decoding* functions, where `<variant> == {…​; #<id>: () → T; …​}` contains one variant per `shared` or `shared query` function, `<id>`, of the actor
+    (`shared composite query` functions are ignored).
```

Comment:
This means that `inspect` is not called for composite queries? Or is the message argument for `shared composite query` simply mapped to `shared query`?

Reply:
Inspect is indeed not called for composite queries (in my understanding) so we don't even include them in the variant type.

Reply:
Just to clarify, even for querie method, inspect is only called if the method is invoked as an (ingress) update call (with consensus). It's not used for the faster query calls (no-concensus). Very confusing.

Since composite query methods can't be called with (ingress) update calls, just (ingress) query calls, there is no need to include them here.